cmake/draco_install.cmake

```diff
@@ -39,19 +38,19 @@ macro(draco_setup_install_target)
   configure_package_config_file(
     "${draco_root}/cmake/draco-config.cmake.template"
     "${draco_build}/draco-config.cmake"
-    INSTALL_DESTINATION "${data_path}/cmake")
+    INSTALL_DESTINATION "${libs_path}/cmake/draco")
```

Comment:
Please revert to data_path, or just cherry-pick the changes from #839 into your branch.

Reply:
solved in merge of master: https://github.com/google/draco/pull/838/commits/85abbb3a22986c7ca7b54b4296f92e15ad877cf9